.github/workflows/cpp-packaging.yml

```diff
@@ -67,8 +65,12 @@ jobs:
         include:
         - os: ubuntu-latest
           tools_platform: linux
+          # Binutils 2.35.1 released Sep 19, 2020
+          binutils_version: "2.35.1"
```

Comment:
Note: I've learned you can't use env substitution in matrix definitions, so I had to move the version numbers directly into here.